FlaxEditor/GUI/Drag/DragHelper.cs

```diff
@@ -80,7 +80,9 @@ protected DragHelper(Func<T, bool> validateFunction)
         /// <returns>Gathered objects or empty IEnumerable if cannot get any valid.</returns>
         public abstract IEnumerable<T> FromDragData(DragData data);
 
+
         //TODO: This could also be an event or something
+        //public DragDropEvent<U> DragDrop;
```

Comment:
Currently, `OnDragDrop` calls `DragDrop`, which can be overriden by child classes. I'm not sure what would be better. A function (currently) or an event.